### PR TITLE
SID added to student application

### PR DESCRIPTION
**Pull request by junomorey**

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# == Schema Information
+ # == Schema Information
```
Comment (warrenshen) [inline app/models/user.rb]: Did you add this space on accident?

**Comment by warrenshen** [comment]
@junomorey @chenhoward should we name the column `student_id` to be more specific?

File: app/models/user.rb (hunk 1 above)
Comment (junomorey) [inline app/models/user.rb]: Oh whoops, yeah

**Comment by junomorey** [comment]
@warrenshen Yeah 'student_id' would probably be better, especially since there's also a 'user_id' column. What about the 'sid' and 'uid' columns in the user table - are those intentionally named differently from the student application table's columns?

**Comment by chenhoward** [comment]
Yeah let's change it to student_id on both users and applications @warrenshen ++ @junomorey

File: app/models/user.rb (hunk 1 above)
Comment (chenhoward) [inline app/models/user.rb]: This space still here doe

Comment (junomorey) [inline app/models/user.rb]: ahhhh man I forgot about that - aight one sec I'll fix it real quick.

File: app/models/student_application.rb [hunk 2]
```diff
@@ -106,4 +109,9 @@ def pend
     pending?
   end
 
+  def update_student_id
```
Comment (chenhoward) [inline app/models/student_application.rb]: Can be changed into
```
def update_student_id
  self.user.update(student_id: self.student_id)
end
```

Comment (junomorey) [inline app/models/student_application.rb]: Ok I'll change that too rn

**Pull request merged**